orttraining/orttraining/python/training/ortmodule/_custom_autograd_function_exporter.py

```diff
@@ -90,7 +90,10 @@ def _export_pt_1_10(g, n, *args, **kwargs):
         if runtime_pytorch_version >= version.parse("1.12"):
             from torch.onnx import _globals
 
-            training_mode = _globals.GLOBALS.training_mode
+            assert _globals.GLOBALS.training_mode == _globals.GLOBALS.training_mode.EVAL \
```

Comment:
yeah as the pylint suggested, lets' use "_globals.GLOBALS.training_mode in (_globals.GLOBALS.training_mode.EVAL, _globals.GLOBALS.training_mode.TRAINING)"

Reply:
done